project.yaml

```diff
@@ -1545,14 +1589,14 @@ Layer:
                   END)
                   *
                   (CASE
-                    WHEN (capital = 'yes') THEN 3
-                    WHEN (capital = '4') THEN 2
+                    WHEN (capital = '4' OR (capital = 'yes' AND admin_level = '4')) THEN 2
```

Comment:
I removed it because I moved country capitals to their own layer. Only capitals with level 4 are still handled by this.